scripts/verify-commit.js

```diff
@@ -2,27 +2,38 @@
 import pico from 'picocolors'
 import { readFileSync } from 'node:fs'
 import path from 'node:path'
+import { exec } from './utils.js'
 
-const msgPath = path.resolve('.git/COMMIT_EDITMSG')
-const msg = readFileSync(msgPath, 'utf-8').trim()
+main()
 
-const commitRE =
-  /^(revert: )?(feat|fix|docs|dx|style|refactor|perf|test|workflow|build|ci|chore|types|wip|release)(\(.+\))?: .{1,50}/
+async function main() {
+  const { stdout: branch } = await exec('git', ['branch', '--show-current'])
+  // only verify commit message on main and minor branches
+  if (branch !== 'main' && branch !== 'minor') {
```

Comment:
@yyx990803 so, only verify on main and minor branches

Reply:
I fully understand the intention behind this PR. However, I want to emphasize that verifying commit messages is not a burden for contributors, as they are already aware of the content of their own messages. Additionally, this practice can help them develop good commit habits, which is particularly beneficial in a collaborative development environment.